v2/pkg/common/reconciler/reconcile_policy.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package reconciler
+
+type ReconcilePolicy string
+
+const (
+	// ReconcilePolicyManage instructs the operator to manage the resource in question.
+	// This includes issuing PUTs to update it and DELETE's to delete it from Azure if deleted in Kubernetes.
+	// This is the default policy when no policy is specified.
+	ReconcilePolicyManage = ReconcilePolicy("manage")
```

Comment:
The common prefix here mirrors the name of the package, so can probably be removed for the type and all values.

```suggestion
	PolicyManage = Policy("manage")
```